packages/Webkul/Admin/src/Resources/views/users/forget-password/create.blade.php

```diff
@@ -7,10 +7,11 @@
     <div class="flex justify-center items-center h-[100vh]">
         <div class="flex flex-col gap-5 items-center">
             <!-- Logo -->
-            <img 
-                class="w-max" 
-                src="{{ bagisto_asset('images/logo.svg') }}" 
-                alt="Bagisto Logo"
+            <img
+                src="{{ core()->getCurrentChannel()->logo_url ?? bagisto_asset('images/logo.svg') }}"
```

Comment:
Admin is something that is not a channel. You need to use configuration, you can go to the 'Configure -> Design' and there you will see the admin logo, just place that key in the below method,
core()->getConfigData('...')